### PR TITLE
Add a test to publish workflow

### LINKED ISSUE #80: The publish workflow doesn't test the images before publishing them
Currently, the latest image of pulp is broken. Have the publish workflow test the images before publishing them.

### PR DESCRIPTION
**Pull request by daviddavis**
fixes #80

### CODE REVIEW

**Review by fao89 — APPROVED** [review]
sweet!

**Comment by bmbouter** [comment]
This looks good to me! I'll defer lgtm's to the folks who work with this more than I do. Thanks @daviddavis !

File: assets/test.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+for _ in $(seq 10)
+do
+  sleep 3
+  if curl --fail http://localhost:8080/pulp/api/v3/status/ > /dev/null 2>&1
+  then
+    break
+  fi
+done
+curl --fail http://localhost:8080/pulp/api/v3/status/
+echo "SUCCESS: $image passed."
```
Comment (mdellweg) [inline assets/test.sh]: `docker rm pulp`
`stop` or `halt` may be needed first. But should we not free the name pulp for the next container?

Comment (daviddavis) [reply assets/test.sh]: I've tried to keep it simple by only running one container.

Comment (daviddavis) [reply assets/test.sh]: Added stop and rm in a cleanup trap.

**Review by mdellweg — APPROVED** [review]

File: assets/test.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+cleanup() {
+  docker stop pulp
+  docker rm pulp
```
Comment (mdellweg) [inline assets/test.sh]:
```suggestion
```
Sorry, I was the one asking for it, but by `rm`ing that container, we loose it's logs. XD

**Review by mdellweg — CHANGES_REQUESTED** [review]

**Review by mdellweg — APPROVED** [review]

**Comment by fao89** [comment]
CI is happy now!

**Pull request merged**